tw-tkms-starter/src/test/java/com/transferwise/kafka/tkms/MonitoringIntTest.java

```diff
@@ -12,37 +13,37 @@ class MonitoringIntTest extends BaseIntTest {
   @Test
   void testThatMonitoringMetricsArePresent() {
     Awaitility.await().until(() -> {
-      Gauge gauge = meterRegistry.find("tw.tkms.dao.approximate.messages.count").tags("shard", "0", "partition", "0").gauge();
+      Gauge gauge = meterRegistry.find(TkmsMetricsTemplate.DAO_APPROXIMATE_MESSAGES_COUNT).tags("shard", "0", "partition", "0").gauge();
```

Comment:
I think the problem replacing it with actual constant is that the test no longer tests backwards compatibility. Basically if you change the metric name then the test will pass, but for consumers it will actually be a breaking change.